src/mappings/UController/mapping.ts

```diff
@@ -174,17 +175,12 @@ export function handleRegisterScheme(event: RegisterScheme): void {
   domain.handleRegisterScheme(event.params._avatar, org.value0 , org.value1, event.params._scheme, paramsHash);
 
   // Detect a new organization event by looking for the first register scheme event for that org.
-  let isFirstRegister = store.get(
-    'FirstRegisterScheme',
-    event.params._avatar.toHex(),
-  );
+  let isFirstRegister = FirstRegisterScheme.load(event.params._avatar.toHex());
   if (isFirstRegister == null) {
     insertOrganization(event.address, event.params._avatar);
-    store.set(
-      'FirstRegisterScheme',
-      event.params._avatar.toHex(),
-      new Entity(),
-    );
+    isFirstRegister = new FirstRegisterScheme(event.params._avatar.toHex());
+    isFirstRegister.id = event.params._avatar.toHex();
```

Comment:
this line is redundant as the id is set be default at line 181.

Reply:
removed